code/modules/library/lib_items.dm

```diff
@@ -283,6 +284,67 @@
 	else
 		return ..()
 
+/obj/item/weapon/book/mindbook
+	name = "A Harmless Book"
+	icon = 'icons/obj/library.dmi'
+	icon_state ="book"
+	unique = 1
+	forbidden = 1
+	var/hallucinatory_cooldown
+
+/obj/item/weapon/book/mindbook/attack_self(mob/living/H)

```

Comment:
`/obj/item/weapon/book/mindbook/attack_self(mob/living/user)`

is the standard.